### ISSUE TITLE
Unexpected behavior when the current directory no longer exist

### ISSUE THREAD

**Issue by sebdej** [PowerShell/PowerShell#18663]
### Prerequisites

- [X] Write a descriptive title.
- [X] Make sure you are able to repro it on the [latest released version](https://github.com/PowerShell/PowerShell/releases)
- [X] Search the existing issues.
- [X] Refer to the [FAQ](https://github.com/PowerShell/PowerShell/blob/master/docs/FAQ.md).
- [X] Refer to [Differences between Windows PowerShell 5.1 and PowerShell](https://docs.microsoft.com/powershell/scripting/whats-new/differences-from-windows-powershell).

### Steps to reproduce

When the current directory no longer exists, if I invoke a command (not a cmdlet), the command is executed silently in another window.

### Expected behavior

```console
It should be invoked in the current window or should display an error message.
```

### Actual behavior

```console
The command is executed in a flashing window.
```

### Environment data

```powershell
Name                           Value
----                           -----
PSVersion                      7.2.7
PSEdition                      Core
GitCommitId                    7.2.7
OS                             Microsoft Windows 10.0.22000
Platform                       Win32NT
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0…}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0
```

### Visuals

https://user-images.githubusercontent.com/2871681/204003012-513be091-82a3-435a-ad23-723159727617.mp4

**Comment by 237dmitry** [comment]
> if I invoke a command (not a cmdlet), the command is executed silently in another window.

I think it depends on terminal emulator. For example, in Windows Terminal cmd.exe launched in the same tab with the path of user home directory, because could not find current. In the prompt of cmd.exe is `OSC 9;9` to get current path from console. In conhost cmd.exe launches with `$pshome` path:

![Screenshot 2022-11-25 214921](https://user-images.githubusercontent.com/78153320/204041592-28e46789-76fe-480b-990f-6f12f8a2ddd8.png)

**Comment by SteveL-MSFT** [comment]
I believe this was fixed in 7.3
